service.go

```diff
@@ -542,6 +542,20 @@ func (s *Service) GetMempoolPendingOperations(ctx context.Context, chainID strin
 	return &ops, nil
 }
 
+// MonitorMempoolOperations monitors mempool pending operations.
```

Comment:
It's worth mentioning in the doc comment that the channel will close after every block, requiring the caller to reconnect.